### PR TITLE
Make `appendToBody` work in Tooltip

### PR DESCRIPTION
**Pull request by jtomaszewski**
If almost everything is already implemented, then why not just make this work?

### CODE REVIEW

**Comment by adrianfaciu** [comment]
Can you please fix the lint issue.

**Comment by jtomaszewski** [comment]
> Can you please fix the lint issue.

Fixed.

**Comment by valorkin** [comment]
@jtomaszewski is positioning working properly?
can you please add sample to demo page?

**Comment by valorkin** [comment]
Thanks a lot for your help :)

**Comment by jtomaszewski** [comment]
> @jtomaszewski is positioning working properly?

Yes.

> can you please add sample to demo page?

Done ;)

File: components/tooltip/tooltip.directive.ts [hunk 1]
```diff
@@ -64,6 +65,10 @@ export class TooltipDirective {
     this.tooltip = this.componentsHelper
       .appendNextToLocation(TooltipContainerComponent, this.viewContainerRef, binding);
 
+    if (this.appendToBody) {
+      this.componentsHelper.getDocument().body.appendChild(this.tooltip.location.nativeElement);
```
Comment (valorkin) [inline components/tooltip/tooltip.directive.ts]: can you use appendNextToRoot instead of appendChild?

**Review by valorkin — CHANGES_REQUESTED** [review]
ok, this review tool is completely new to me :)
please try to use appendNextToRoot
or it will not work from universal

File: components/tooltip/tooltip.directive.ts (hunk 1 above)
Comment (jtomaszewski) [reply components/tooltip/tooltip.directive.ts]: Sure. Anything else I should take care about? ;)

Comment (valorkin) [reply components/tooltip/tooltip.directive.ts]: no, I suppose this one is the main one

Comment (valorkin) [reply components/tooltip/tooltip.directive.ts]: you can check how it is done in tooltips for example

**Comment by jtomaszewski** [comment]
I changed it to use `appendNextToRoot` method , squashed it into one commit and rebased to current development branch. IMHO it's good to merge ;)

**Comment by valorkin** [comment]
This is awesome! Thank you!

**Comment by DethAriel** [comment]
@valorkin are there any obstacles yet for merging this into master?

**Review by valorkin — APPROVED** [review]

**Comment by valorkin** [comment]
Not at all, just stuck with AoT and routing a bit

**Pull request merged**